hnn_core/gui/_viz_manager.py

```diff
@@ -937,6 +944,11 @@ def data(self):
             "figs": self.figs
         }
 
+    @property
+    def fig_default_params(self):
```

Comment:
Does this need to be `@property`? Seems like the other ones were created to regroup existing attributes for code cleanliness, whereas here you're just returning `self.fig_defaults`

Reply:
I made it a @property so I could call it in the test:
```viz_smooth_value = gui.viz_manager.fig_default_params['default_smoothing']```